metric-pro-stats/auto-update-stats

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -uxe
+# This meant to run each Sun->Mon night checking what it might need to update
+[[ "$(LC_ALL=C date +%A)" == "Monday" ]] || exit 1
+echo "Sanity check passed, this is a Monday"
+
+path="$(dirname "$0")"
+defaultoptions="--log info --cachefile /tmp/esm-stats-new.cache --append --connectionenv ~/work/credentials/esm-stats/ps5-prod-esm-cache-ro.novarc ~/work/credentials/esm-stats/ps4.5-prod-esm-cache-ro.novarc"
```

Comment:
Let's not hardcode /tmp and give precedence to $TMPDIR if set (so we'll use a job specific tmpdir in Jenkins). Also that `~work/` path won't work in Jenkins, but I guess that's part of the WIP state.

Reply:
Now I feel bad `auto-update-stats` is only an artifact of some former local experiments. That is now properly split and much better in the yaml. I'll remove it - sorry for the noise.